### DISCUSSION TITLE
PTZ Usage Options

### DISCUSSION THREAD

**Discussion by ropeguru** [blakeblackshear/frigate#5464] [General]
So I have setup Frigate a couple of times, but one of the things I am trying to deal with are cameras with PTZ. 

While not directly related to Frigate usage, what are folks using for PTZ outside of Frigate? Looking for something which has decent integration and usage with a HA and Frigate deployment.

**Comment by NickM-27** [comment]
I was using HA but I am now working on adding PTZ controls to frigate directly https://github.com/blakeblackshear/frigate/pull/4715
